pyqtgraph/__init__.py

```diff
@@ -55,7 +55,7 @@
                                  # change in the future.
     'useCupy': False,  # When True, attempt to use cupy ( currently only with ImageItem and related functions )
     'useNumba': False, # When True, use numba
-    'segmentedLineMode': 'auto',  # segmented line mode, controls if lines are plotted in segments or continuous
```

Comment:
Does this need to be reverted before we can merge?

Reply:
Yes, it does. But as the release of PySide6 6.4.3 is imminent, I thought I would just wait to test out against the official release and fix up the comments regarding working version numbers.

Reply:
Oh right, keep forgetting 6.4.3 hasn't been released yet!

Reply:
The 6.4.3 wheels are now available at https://download.qt.io/snapshots/ci/pyside/6.4/latest/split_wheels/ but not yet on pypi.org